LICENSE

```diff
@@ -1,201 +1,9 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+The MIT License (MIT)
```

Comment:
@mattsayar-splunk 
Contributor has updated the License to MIT and also update permission verbose on line #5, is it okay or we need to revert license back to MIT license

Reply:
Hi @asingh-bitglass 

Can you please help us to understand why the License has been changed from Apache License to MIT ?

Reply:
Keeping MIT license as per discussion with Matt